client/components/StartPopUp/StartPopUp.tsx

```diff
@@ -1,27 +1,26 @@
 'use client';
 
 import React, { useState } from 'react';
-
 import Agreement from 'components/Agreement/Agreement';
 import Condition from 'components/Condition/Condition';
```

Comment:
If these components are used only here, then let's immediately put them in the folder with this component